### PR TITLE
Use FRAME_TYPE enum instead of plain int.

### PR DESCRIPTION
**Pull request by kronenpj**
Move to using the statoc.FRAME_TYPE enumeration instead of integers for frame types in data_handler.py.
Update FRAME_TYPE enum for missing items.

### CODE REVIEW

File: tnc/data_handler.py [hunk 1]
```diff
@@ -261,92 +272,100 @@ def process_data(self, bytes_out, freedv, bytes_per_frame: int) -> None:
...
             # ARQ CHANNEL IS OPENED
-            elif frametype in [226, 228]:
+            elif frametype in [
+                FR_TYPE.ARQ_DC_OPEN_ACK_W.value,
+                FR_TYPE.ARQ_DC_OPEN_ACK_N.value,
+            ]:
                 self.log.debug("[TNC] ARQ arq_received_channel_is_open")
                 self.arq_received_channel_is_open(bytes_out[:-2])
 
             # ARQ MANUAL MODE TRANSMISSION
```
Comment (DJ2LS) [inline tnc/data_handler.py:350]: In my opinion we can remove the manual mode selection, as a automatic selection fits best, specially under really bad HF conditions. This feature works well in development environments but not under real circumstances

File: tnc/data_handler.py [hunk 2]
```diff
@@ -413,7 +432,7 @@ def send_data_to_socket_queue(self, /, **jsondata):
     def send_burst_ack_frame(self, snr) -> None:
         """Build and send ACK frame for burst DATA frame"""
         ack_frame = bytearray(14)
-        ack_frame[:1] = bytes([60])
+        ack_frame[:1] = bytes([FR_TYPE.ACK.value])
```
Comment (DJ2LS) [inline tnc/data_handler.py:435]: This one should be named BURST_ACK so it's homogeneous to the other names.

File: tnc/static.py [hunk 3]
```diff
@@ -161,6 +148,9 @@ class FRAME_TYPE(Enum):
     ARQ_DC_OPEN_ACK_W = 226
     ARQ_DC_OPEN_N = 227
     ARQ_DC_OPEN_ACK_N = 228
+    ARQ_MANUAL_01 = 230
+    # ...
```
Comment (DJ2LS) [inline tnc/static.py:152]: Same thought here with removing like above 🤔

**Review by DJ2LS — APPROVED** [review]
Approved by checking from mobile phone at work. Wasn't able testing this by running the application.

File: tnc/static.py (hunk 3 above)
Comment (kronenpj) [reply tnc/static.py]: The latest push removes the manual mode code.

File: tnc/data_handler.py (hunk 2 above)
Comment (kronenpj) [reply tnc/data_handler.py]: Done.

**Review by DJ2LS — APPROVED** [review]

**Pull request merged**